matter_server/common/custom_clusters.py

```diff
@@ -243,6 +245,52 @@ def attribute_type(cls) -> ClusterObjectFieldDescriptor:
 
             value: float32 = 0
 
+        @dataclass
+        class Altitude(ClusterAttributeDescriptor, CustomClusterAttributeMixin):
+            """Altitude Attribute within the Eve Cluster."""
+
+            should_poll = False
+
+            @ChipUtility.classproperty
+            def cluster_id(cls) -> int:
+                """Return cluster id."""
+                return 0x130AFC01
+
+            @ChipUtility.classproperty
+            def attribute_id(cls) -> int:
+                """Return attribute id."""
+                return 0x130A0013
+
+            @ChipUtility.classproperty
+            def attribute_type(cls) -> ClusterObjectFieldDescriptor:
+                """Return attribute type."""
+                return ClusterObjectFieldDescriptor(Type=int)
+
+            value: int = 0
+
+        @dataclass
+        class Pressure(ClusterAttributeDescriptor, CustomClusterAttributeMixin):
+            """Pressure Attribute within the Eve Cluster."""
+
+            should_poll = True
```

Comment:
As discussed in the other review, this does not need to be polled. The default is `False`, you can just drop it:

```suggestion
```